docs/source/savant_101/10_adapters.rst

```diff
@@ -833,6 +833,7 @@ The simplified design of the adapter is depicted in the following diagram:
 - ``SYNC_OUTPUT``: a flag indicates whether to show frames on sink synchronously (i.e. at the source rate); the streaming may be not stable with this flag, try to avoid it; the default value is ``False``;
 - ``SOURCE_ID``: a filter to receive frames with a specific ``source_id`` only. The sink works in single-stream mode when this option is specified;
 - ``SOURCE_IDS``: a filter to receive frames with specific ``source_id``-s only. The sink works in multi-stream mode when this option is specified.
+- ``MAX_ALLOWED_RESOLUTION``: Maximum resolution of the incoming stream. If the resolution is greater than the allowed resolution, the adapter will terminate. You can override the max allowed resolution be setting width and height of frames. The default value is ``3840x2152``.
```

Comment:
The adapter won't terminate itself. I don't think it should. By design the adapter always works and shows either video stream or stub image.
In this case the adapter will show an error, terminate and restart the input pipeline.

Reply:
```
 ERROR insight::savant::savant_rs_video_demux                       > The resolution of the incoming stream is 1280x720 and treater than the allowed max 100x100 resolutions. Terminate. You can override the max allowed resolution with 'MAX_ALLOWED_RESOLUTION' environment variable.
 ERROR insight::savant::gstreamer::runner                           > Received error "gst-stream-error-quark: The resolution of the incoming stream is 1280x720 and treater than the allowed max 100x100 resolutions. Terminate. You can override the max allowed resolution with 'MAX_ALLOWED_RESOLUTION' environment variable. (9)" from savant_rs_video_demux+savantrsvideodemux6.
Exception in thread Thread-22:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.8/dist-packages/savant/gstreamer/runner.py", line 75, in _main_loop_run
    raise RuntimeError(self._error)
RuntimeError: Received error "gst-stream-error-quark: The resolution of the incoming stream is 1280x720 and treater than the allowed max 100x100 resolutions. Terminate. You can override the max allowed resolution with 'MAX_ALLOWED_RESOLUTION' environment variable. (9)" from savant_rs_video_demux+savantrsvideodemux6. Debug info: "/opt/savant/gst_plugins/python/savant_rs_video_demux.py(561): check_max_allowed_resolution (): /GstPipeline:input-pipeline/savant_rs_video_demux+SavantRsVideoDemux:savant_rs_video_demux+savantrsvideodemux6".
```

Reply:
@bwsw Should we terminate the adapter or interrupt the display of the input video stream but show the stub image?

Reply:
Modified